test/test-e2e.sh

```diff
@@ -77,6 +78,9 @@ setup_kubectl() {
 
 create_cluster() {
   KIND_CONFIG="$PWD/test/kind-config.yaml"
+  if [[ ${NEED_ENABLE_FEATURE} == "true" ]]; then
+  KIND_CONFIG="$PWD/test/kind-config1.28.yaml"
+  fi
```

Comment:
@yangjunmyfm192085 this comment still applies. I was suggesting adding a top-level variable like the following one instead of `NEED_ENABLE_FEATURE` to avoid branching:
```
${KIND_CONFIG:="$PWD/test/kind-config.yaml"}
```
We can then set it in the makefile when invoking the rules